admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -154,6 +154,27 @@ x users found:
   This is an indicator that tells you approximately how many users will be listed in ownCloud.
   The number updates automatically after any changes.
 
+  Active Directory offers "Recursive retrieval of all AD group memberships of a user". This means essentially that you would be able to search the group you enter and all the other child groups from this groups for users.
+
+  Enter this filter to access this feature::
+
+    ``(&(objectClass=user)(memberof:1.2.840.113556.1.4.1941:=CN=<Group_Name>,DC=example,DC=com))``
+
+  Enter your group name instead of the "<Group_Name>" placeholder.
+
+  If you want to search multiple groups with this feature adjust your filter like this:
+
+    ``"(&(objectClass=user)(|(memberOf:1.2.840.113556.1.4.1941:=CN=<groupname>,CN=Users,DC=example,DC=com)(memberOf:1.2.840.113556.1.4.1941:=CN=<group2name>,CN=Users,DC=example,DC=com)))"``
+
```

Comment:
Makes it much clearer.
How is the notation when you have n-groups ?
```
(&(objectClass=user)(|
  (memberOf:....)
  (memberOf:....)
  (memberOf:....)
  ...
))
```
Is it allowed to write in multiple lines, respectively have blanks between brackets which would make readablity better?
If my thoughts are right, I propose to first write for multiple groups the general notation (my example if that works) and then an example as you did (in the formatting of the general multi group example)

groupname --> groupname1
group2name --> groupname2

Reply:
groupname --> groupname1
group2name --> groupname2

This makes the alignment of all elements correct and looks professional

Before
```
(memberOf:1.2.840.113556.1.4.1941:=CN=<groupname>,CN=Users,DC=example,DC=com)
(memberOf:1.2.840.113556.1.4.1941:=CN=<group2name>,CN=Users,DC=example,DC=com)
```
After
```
(memberOf:1.2.840.113556.1.4.1941:=CN=<groupname1>,CN=Users,DC=example,DC=com)
(memberOf:1.2.840.113556.1.4.1941:=CN=<groupname2>,CN=Users,DC=example,DC=com)
```

And I have to ask, you can add more lines with a single ``|`` (or) statement ?
If yes, please add a text statement below that you can add multiple lines, one for each group

BTW is it allowed to have this block in double ticks instead of triple as we would do for a block usually ?